project/PekkoDependency.scala

```diff
@@ -44,6 +44,7 @@ object PekkoDependency {
       case None =>
         Option(System.getProperty("pekko.http.build.pekko.version")) match {
           case Some("main")           => mainSnapshot
+          case Some("snapshot-1.0.x") => snapshot10x
```

Comment:
Can we change the naming to `main-1.0.x` so its consistent with `main`?

Reply:
`main` is the name of the branch we are using for 1.1.x 
so calling the option to use a 1.0.x snapshot something with the word `main` in it would be very confusing

Reply:
> `main` is the name of the branch we are using for 1.1.x so calling the option to use a 1.0.x snapshot something with the word `main` in it would be very confusing

No its not, `main` points to the latest `main` branch which are snapshots. There is an argument the naming is confusing but if we want to change that lets do it in another PR